Jenkinsfile

```diff
@@ -140,149 +128,92 @@ pipeline {
             }
             steps {
-                script {
-                    STAGE = "Deploy"
-                }
                 milestone(1)
                 lock('CH Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${CH_TABLE}-${MODULE_NAME} deployment in progress")
+                    colourText("info", "${env.DEPLOY_TO}-${CH_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(CH_TABLE, false)
-                    colourText("success", "${env.DEPLOY_NAME}-${CH_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${CH_TABLE}-${MODULE_NAME} Deployed.")
                 }
                 lock('VAT Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${VAT_TABLE}-${MODULE_NAME} deployment in progress")
+                    colourText("info", "${env.DEPLOY_TO}-${VAT_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(VAT_TABLE, false)
-                    colourText("success", "${env.DEPLOY_NAME}-${VAT_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${VAT_TABLE}-${MODULE_NAME} Deployed.")
                 }
                 lock('PAYE Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${PAYE_TABLE}-${MODULE_NAME} deployment in progress")
+                    colourText("info", "${env.DEPLOY_TO}-${PAYE_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(PAYE_TABLE, false)
-                    colourText("success", "${env.DEPLOY_NAME}-${PAYE_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${PAYE_TABLE}-${MODULE_NAME} Deployed.")
                 }
-		        lock('Legal Unit Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${LEU_TABLE}-${MODULE_NAME} deployment in progress")
+                lock('Legal Unit Deployment Initiated') {
+                    colourText("info", "${env.DEPLOY_TO}-${LEU_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(LEU_TABLE , true)
-                    colourText("success", "${env.DEPLOY_NAME}-${LEU_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${LEU_TABLE}-${MODULE_NAME} Deployed.")
                 }
             }
-        }
-
-        stage ('Package and Push Artifact') {
-            agent any
-            when {
-                expression {
-                    isBranch(BRANCH_DEV) || isBranch(BRANCH_TEST)
+            post {
+                success {
+                    colourText("info","Stage: ${env.STAGE} successful!")
                 }
-            }
-            steps {
-                script {
-                    env.NODE_STAGE = "Package and Push Artifact"
+                failure {
+                    colourText("warn","Stage: ${env.STAGE} failed!")
```

Comment:
Maybe in the future we could capture which API fails here to make it easier to the user.

Reply:
We would have to script that behaviour. Lets catch it in a Jira.